Fixing "Additional settings" link

ryancrawcour opened the pull request:
Additional settings link was linking to a GitHub document, which says that the file is not being maintained and that you should go to some other document on docs.microsoft.com. So fixed now to just go directly to the correct document in the first place.

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

ryancrawcour committed:
Fixing "Additional settings" link

Additional settings link was linking to a GitHub document, which says that the file is not being maintained and that you should go to some other document on docs.microsoft.com. So fixed now to just go directly to the correct document in the first place.

GitHubber17 committed:
Merge branch 'master' into patch-1
